hadml/models/components/encoder_transformer.py

```diff
@@ -0,0 +1,67 @@
+import torch, os, pickle
+
+
+class Generator(torch.nn.Module):
+    """ Generator implemented as a encoder-only transformer model """
+
+    def __init__(
+        self,
+        noise_dim=4,            # Arbitrary number (noise dimensionality)
+        cluster_kins_dim=8,     # Cluster four-momentum, two quark types, phi, theta
```

Comment:
Does it make sense to have quark types included in this kinematics variable? Wouldn't `cluster_kins_dim=6` be better?